win32ss/w32ksvc64.h

```diff
@@ -0,0 +1,690 @@
+    SVC_(UserLockWorkStation, 0)	//	�0x125e�
+    SVC_(UserMNDragLeave, 0)	//	�0x125f�
+    SVC_(UserMNDragOver, 2)	//	�0x1260�
+    SVC_(UserMenuItemFromPoint, 4)	//	�0x1261�
+    SVC_(UserMinMaximize, 3)	//	�0x1262�
+    SVC_(UserNotifyIMEStatus, 3)	//	�0x1263�
+    SVC_(UserOpenInputDesktop, 3)	//	�0x1264�
+    SVC_(UserPrintWindow, 3)	//	�0x1265�
+    SVC_(UserQueryInformationThread, 4)	//	�0x1266�
+    SVC_(UserQueryInputContext, 2)	//	�0x1267�
+    SVC_(UserQuerySendMessage, 1)	//	�0x1268�
+    SVC_(UserRealChildWindowFromPoint, 3)	//	�0x1269�
+    SVC_(UserRealWaitMessageEx, 2)	//	�0x126a�
+    SVC_(UserRegisterHotKey, 4)	//	�0x126b�
+    SVC_(UserRegisterRawInputDevices, 3)	//	�0x126c�
+    SVC_(UserRegisterTasklist, 1)	//	�0x126d�
+    SVC_(UserRegisterUserApiHook, 4)	//	�0x126e�
+    SVC_(UserRemoteConnect, 3)	//	�0x126f�
+    SVC_(UserRemoteRedrawRectangle, 4)	//	�0x1270�
+    SVC_(UserRemoteRedrawScreen, 0)	//	�0x1271�
+    SVC_(UserRemoteStopScreenUpdates, 0)	//	�0x1272�
+    SVC_(UserResolveDesktopForWOW, 1)	//	�0x1273�
+    SVC_(UserSetAppImeLevel, 2)	//	�0x1274�
+    SVC_(UserSetClassWord, 3)	//	�0x1275�
+    SVC_(UserSetCursorContents, 2)	//	�0x1276�
+    SVC_(UserSetImeHotKey, 5)	//	�0x1277�
+    SVC_(UserSetImeInfoEx, 1)	//	�0x1278�
+    SVC_(UserSetImeOwnerWindow, 2)	//	�0x1279�
+    SVC_(UserSetInternalWindowPos, 4)	//	�0x127a�
+    SVC_(UserSetLayeredWindowAttributes, 4)	//	�0x127b�
+    SVC_(UserSetLogonNotifyWindow, 1)	//	�0x127c�
+    SVC_(UserSetMenu, 3)	//	�0x127d�
+    SVC_(UserSetMenuContextHelpId, 2)	//	�0x127e�
+    SVC_(UserSetMenuFlagRtoL, 1)	//	�0x127f�
+    SVC_(UserSetObjectInformation, 4)	//	�0x1280�
+    SVC_(UserSetShellWindowEx, 2)	//	�0x1281�
+    SVC_(UserSetSysColors, 4)	//	�0x1282�
+    SVC_(UserSetSystemCursor, 2)	//	�0x1283�
+    SVC_(UserSetSystemTimer, 4)	//	�0x1284�
+    SVC_(UserSetThreadLayoutHandles, 2)	//	�0x1285�
+    SVC_(UserSetWindowStationUser, 4)	//	�0x1286�
+    SVC_(UserSoundSentry, 0)	//	�0x1287�
+    SVC_(UserSwitchDesktop, 1)	//	�0x1288�
+    SVC_(UserTestForInteractiveUser, 1)	//	�0x1289�
+    SVC_(UserTrackPopupMenuEx, 6)	//	�0x128a�
+    SVC_(UserUnloadKeyboardLayout, 1)	//	�0x128b�
+    SVC_(UserUnlockWindowStation, 1)	//	�0x128c�
+    SVC_(UserUnregisterHotKey, 2)	//	�0x128d�
+    SVC_(UserUnregisterUserApiHook, 0)	//	�0x128e�
+    SVC_(UserUpdateInputContext, 3)	//	�0x128f�
+    SVC_(UserUpdateInstance, 3)	//	�0x1290�
+    SVC_(UserUpdateLayeredWindow, 10)	//	�0x1291�
+    SVC_(UserUpdatePerUserSystemParameters, 2)	//	�0x1292�
+    SVC_(UserUserHandleGrantAccess, 3)	//	�0x1293�
+    SVC_(UserValidateHandleSecure, 1)	//	�0x1294�
+    SVC_(UserWaitForInputIdle, 3)	//	�0x1295�
+    SVC_(UserWaitForMsgAndEvent, 1)	//	�0x1296�
+    SVC_(UserSetClassLongPtr, 0)	//	�0x1297�
+    SVC_(UserSetWindowLongPtr, 4)	//	�0x1298�
+    SVC_(UserWin32PoolAllocationStats, 6)	//	�0x1299�
+    SVC_(UserYieldTask, 0)	//	�0x129a�
+    SVC_(DxEngGetRedirectionBitmap, 1)	//	
```

Comment:
What are those extra syscalls? Some that exist in win32k debug versions? 
(Also, just noticed that there are "strange" characters around those hex numbers...)

Reply:
Those extra syscalls are either debug functions or just made up reactos stuff. We had plenty of that stuff before we started to actually get our syscall interface aligned with Windows. And the "strange" characters are tabs from exporting from libre office calc.